lookout/style/format/analyzer.py

```diff
@@ -588,29 +532,21 @@ def _group_line_nodes(
                    (line_y, line_y_pred, line_vnodes_y, line_vnodes, line_rule_winners))
 
     @classmethod
-    def _load_analyze_config(cls, config: Mapping[str, Any]) -> Mapping[str, Any]:
+    def _load_config(cls, config: Mapping[str, Any]) -> Mapping[str, Any]:
         """
-        Merge config for `analyze()` with the default config.
+        Merge provided config with the default values.
 
         :param config: User-defined config.
         :return: Full config.
         """
-        return merge_dicts(cls.defaults_for_analyze, config)
-
-    @classmethod
-    def _load_train_config(cls, config: Mapping[str, Any]) -> Mapping[str, Any]:
-        """
-        Merge config for `train()` with the default config.
-
-        :param config: User-defined config.
-        :return: Full config.
-        """
-        config = merge_dicts(cls.defaults_for_train, config)
-        global_config = config.pop("global")
-        try:
-            return {lang: merge_dicts(global_config, lang_config)
-                    for lang, lang_config in config.items()}
-        except AttributeError as e:
-            raise ValueError("Config %s can not be merged with default values config: %s: %s" % (
-                config, global_config, e,
-            )) from None
+        effective_config = merge_dicts(cls.default_config, config)
+        for key in ["train", "analyze"]:
+            global_config = effective_config[key].pop("language_defaults")
+            try:
+                for lang in CONFIGURABLE_LANGUAGES:
```

Comment:
Let's avoid yet another pointer to the languages that we support. Try to do it without this variable by enforcing additional constraints.